crates/aptos/src/move_tool/aptos_debug_natives.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_types::account_config::CORE_CODE_ADDRESS;
+use move_vm_runtime::native_functions::NativeFunctionTable;
+
+pub fn aptos_debug_natives() -> NativeFunctionTable {
+    move_stdlib::natives::all_natives(CORE_CODE_ADDRESS)
```

Comment:
Took me a little while to realize what's the difference between this and `aptos_natives()`. 😂 
Would be nice if a comment is here to make it explict that this `move_stdlib` has the `testing` feature enabled.